templates/hpc/index.html

```diff
@@ -0,0 +1,717 @@
+                url="https://assets.ubuntu.com/v1/0d09fbe7-pgs-logo.png",
+                alt="",
+                width="288",
+                height="288",
+                hi_def=True,
+                loading="auto",
+              attrs={"class": "p-logo-section__logo"}
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{
+                image (
+                url="https://assets.ubuntu.com/v1/ef528144-astrazeneca-logo.png",
+                alt="",
+                width="288",
+                height="288",
+                hi_def=True,
+                loading="auto",
+              attrs={"class": "p-logo-section__logo"}
+            ) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-8">
+        <div>
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/f880a3bd-Enterprise+support.svg",
+            alt="",
+            width="80",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <hr class="u-sv1" />
+        <h2>HPC is used across multiple sectors and industries</h2>
+        <p>
+          From energy to healthcare, we help industry leaders ease operations and break new ground with the power of compute.
+        </p>
+        <p>Accelerate discovery in:</p>
+      </div>
+      <div class="col-4 u-hide--medium u-hide--small u-vertically-center">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/b2dbf6af-CharmedK8s.svg",
+          alt="",
+          width="188",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+      <div class="p-strip is-shallow">
+        <ul class="p-list is-split">
+          <li class="p-list__item is-ticked">Agriculture</li>
+          <li class="p-list__item is-ticked">Automotive</li>
+          <li class="p-list__item is-ticked">Banking and finance</li>
+          <li class="p-list__item is-ticked">Construction</li>
+          <li class="p-list__item is-ticked">Defence and security</li>
+          <li class="p-list__item is-ticked">Electronics</li>
+          <li class="p-list__item is-ticked">Energy and utilities</li>
+          <li class="p-list__item is-ticked">Government</li>
+          <li class="p-list__item is-ticked">Healthcare and pharmaceuticals</li>
+          <li class="p-list__item is-ticked">Life sciences</li>
+          <li class="p-list__item is-ticked">Manufacturing and industry</li>
+          <li class="p-list__item is-ticked">Media and entertainment</li>
+          <li class="p-list__item is-ticked">Research and universities</li>
+          <li class="p-list__item is-ticked">Weather</li>
+        </ul>
+        <p>
+          <a class="p-button--positive" href="/hpc/use-cases">Use cases&nbsp;&rsaquo;</a>

```

Comment:
I think its better to remove `>`  as it is a button
```suggestion
          <a class="p-button--positive" href="/hpc/use-cases">Use cases</a>

```